README.md

```diff
@@ -21,13 +25,19 @@ This can be done through the NuGet package manager in Visual Studio or by runnin
 dotnet add package Dynatrace.OpenTelemetry.Exporter.Metrics
 ```
 
-This exporter package targets [.NET Standard 2.0](https://docs.microsoft.com/en-us/dotnet/standard/net-standard) and can therefore be included on .NET Core 2.0 and above, as well as .NET Framework 4.6.1 and above.
+This exporter package targets [.NET Standard 2.0](https://docs.microsoft.com/en-us/dotnet/standard/net-standard) and can therefore be included on .NET Core 2.0 and above, as well as .NET Framework 4.6.2 and above.
```

Comment:
FYI, they dropped support for .NET framework `4.6.1` a while ago. So I just updated it here for completeness. https://github.com/open-telemetry/opentelemetry-dotnet/pull/3191